packages/firebase_data_connect/firebase_data_connect/lib/src/common/dataconnect_error.dart

```diff
@@ -18,8 +18,8 @@ part of 'common_library.dart';
 enum DataConnectErrorCode { unavailable, unauthorized, other }
 
 /// Error thrown when DataConnect encounters an error.
-class DataConnectError extends FirebaseException {
-  DataConnectError(this.dataConnectErrorCode, message)
+class DataConnectError<T> extends FirebaseException {
```

Comment:
Is this intended that `DataConnectError` has a type parameter? It doesn't look like it's used anywhere. Should it just be the subclass `DataConnectOperationError` that has a type parameter?